praw/objector.py

```diff
@@ -169,7 +169,7 @@ def objectify(
                     # id.
                     data["json"]["data"]["id"] = data["json"]["data"][
                         "id"
-                    ].partition("_")[2]
+                    ].split("_")[1]
```

Comment:
```suggestion
                    ].split("_", 1)[1]
```

Reply:
Cool, didn't know about the second argument.